dwarfstack.qbs

```diff
@@ -0,0 +1,73 @@
+import qbs
+
+DynamicLibrary {
+    name: "dwarfstack"
+    targetName: "dwarfstack" + (qbs.architecture === "x86_64" ? '64' : '32')
+    Depends { name: "cpp" }
+    Export {
+        Depends { name: "cpp" }
+        cpp.includePaths: "include"
+    }
+    Group {
+        fileTagsFilter: product.type
+        qbs.install: true
+    }
+
+    files: [
+        "src/*.c",
+        "mgwhelp/dwarf_pe.c",
+    ]
+    cpp.windowsApiCharacterSet: "mbcs"
+    cpp.defines: "DWST_SHARED"
+    cpp.dynamicLibraries: [
+        "dbghelp",
+        "gdi32",
+        "stdc++",
+    ]
+    cpp.includePaths: [
+        "include",
+        "mgwhelp",
+        "libdwarf",
+    ]
```

Comment:
Did it work for you without adding "zlib", here?
I guess you already have a zlib.h in your system which gcc can use.

Reply:
Good point :)